authorization.adoc

```diff
@@ -11,7 +11,7 @@ https://oauth.izettle.com
 ### Introduction
 The OAuth API implements the OAuth 2.0 Authorization Framework (https://tools.ietf.org/html/rfc6749[RFC 6749]). For an external integrator, there are two different grant types that are of interest, the authorization code grant and the assertion grant. Both grant types are described below. 
 
-The authorization code grant is to be used for public integrations, where you want to create an integration that can be used by any iZettle merchant, whereas the password grant is to be used when you are creating a private integration, for your own merchant account. Please note that for private integrations, this will probably change in the future and another - simpler - solution will be made available where you can create your own access token directly from your merchant dashboard without using the OAuth API.
+The authorization code grant is to be used for public integrations, where you want to create an integration that can be used by any iZettle merchant, whereas the API Key solution is to be used when you are creating a private integration, for your own merchant account.
```

Comment:
👍 on the change. For clarity I suggest we refer to the assertion grant, rather than API Key, since that's the term introduced in the paragraph just above. API key is introduced further down in the document.

```suggestion
The authorization code grant is to be used for public integrations, where you want to create an integration that can be used by any iZettle merchant, whereas the assertion grant is to be used when you are creating a private integration, for your own merchant account.
```